spec/controllers/v2/users/sessions_controller_spec.rb

```diff
@@ -66,32 +66,38 @@
         context 'first failed attempt' do
           before do
             User.maximum_attempts = 2
-            post :create,
-              params: {user: {
-                name: user.to_param,
-                password: "bad-#{user.password}",
-              }},
-              format: :json
+            queue_adapter.performed_jobs = []
+            perform_enqueued_jobs do
+              post :create,
+                params: {user: {
+                  name: user.to_param,
+                  password: "bad-#{user.password}",
+                }},
+                format: :json
+            end
           end
 
           it 'does not lock the user' do
             expect(user.reload.access_locked?).to be(false)
           end
 
           it 'does not send an email' do
-            expect(UsersMailer.deliveries).to be_empty
```

Comment:
Please don't delete this check. I'd still like to know if all emails have actually been delivered.